src/game/mapitems.h

```diff
@@ -151,7 +160,7 @@ struct CMapItemLayer
 
 struct CMapItemLayerTilemap
 {
-	enum { CURRENT_VERSION=4 };
+	enum { CURRENT_VERSION=5 };
```

Comment:
I don't think the map item version increment is necessary. You are only adding flags to existing fields, so you should be able to assume that the flags are unset for old versions.

Reply:
downgraded 👍